benchmark.h(645): warning C4141: 'inline': used more than once

I am compiling release 1.2.0 with Visual Studio 2017 v15.4.0. I have these compiler warnings:

    1>include\benchmark/benchmark.h(645): warning C4141: 'inline': used more than once
    1>include\benchmark/benchmark.h(648): warning C4141: 'inline': used more than once

Corresponding code:

    BENCHMARK_ALWAYS_INLINE inline State::StateIterator State::begin() {
      return StateIterator(this);
    }
    BENCHMARK_ALWAYS_INLINE inline State::StateIterator State::end() {
      StartKeepRunning();
      return StateIterator();
    }

--- a/include/benchmark/benchmark.h
+++ b/include/benchmark/benchmark.h
@@ -642,10 +642,10 @@ struct State::StateIterator {
   State* const parent_;
 };
 
-BENCHMARK_ALWAYS_INLINE inline State::StateIterator State::begin() {
+inline BENCHMARK_ALWAYS_INLINE State::StateIterator State::begin() {
   return StateIterator(this);
 }
-BENCHMARK_ALWAYS_INLINE inline State::StateIterator State::end() {
+inline BENCHMARK_ALWAYS_INLINE State::StateIterator State::end() {
   StartKeepRunning();
   return StateIterator();
 }

